source/scss/_pf-basic-dropdowns.scss

```diff
@@ -0,0 +1,21 @@
+//
+// Dropdowns
+// --------------------------------------------------
+
+// Kebab dropdown
+.pf-dropdown__kebab {
+  .dropdown-toggle::after {
+    display: none;
+  }
+  .btn-link {
```

Comment:
It can get messy quickly. :( Definitely if they would have been nested, I think they'd be joined by the __ as a block__element relationship. But, is the kebab a child of the pf-dropdown, or is it a variant of a pf-dropdown? So maybe it's actually .pf-dropdown--kebab__btn-link. Which looks backwards but reading it carefully you'd know it's a btn-link within the kebab variant of a .pf-dropdown? WDYT?